getusermedia.html

```diff
@@ -1967,6 +1986,17 @@ <h2>The model: sources, sinks, constraints, and settings</h2>
     For example, an &lt;[^img^]&gt; tag scales down a huge source image of
     1600x1200 pixels to fit in a rectangle defined with
     <code>width="400"</code> and <code>height="300"</code>.</p>
+    <p>Sources have a lifetime. By default, a source lifetime is tied
+    to the context that created it. For instance, sources created
+    by {{MediaDevices.getUserMedia()}} are considered as created
+    by its <a href="#dom-navigator-mediadevices">navigator.mediaDevices</a> context.
```

Comment:
I had missed that link (which I now understand was your motivation for the new id.
```suggestion
    by its {{Navigator.mediaDevices}} context.
```
or ({{Navigator/mediaDevices | navigator.mediaDevices }} if the capitalization is an issue